support/docker/dev/janitor.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "PeerTube",
+  "description": "Federated (ActivityPub) video streaming platform using P2P (BitTorrent) directly in the web browser with WebTorrent and Angular.",
+  "icon": "https://janitor.technology/img/peertube.svg",
+  "docker": {
+    "image": "chocobozzz/peertube-dev"
+  },
+  "ports": {
+    "22": {
+      "label": "SSH",
+      "proxy": "none"
+    },
+    "8088": {
+      "label": "VNC",
+      "proxy": "https"
+    },
+    "8089": {
+      "label": "Cloud9",
+      "proxy": "https"
+    },
+    "9000": {
+      "label": "PeerTube",
+      "proxy": "https",
+      "preview": true
+    }
+  },
+  "scripts": {
+    "Start PeerTube": "npm start",
```

Comment:
If https -> `NODE_CONFIG='{ "webserver": { "https": true } }' npm start` should work

Reply:
Thanks, but it's better if PeerTube starts in HTTP. The port won't be accessible publicly, to protect developer containers from unwanted visitors, and requests will only be routed there by our authenticated proxy.